litho-core/src/main/java/com/facebook/litho/LogTreePopulator.java

```diff
@@ -26,18 +26,34 @@
 public final class LogTreePopulator {
   private LogTreePopulator() {}
 
+    /**
+     * Annotate a log event with the log tag set in the context, and extract the treeprops from a
+     * given {@link ComponentContext} and convert them into perf event annotations using a {@link
+     * ComponentsLogger} implementation.
+     *
+     * @return Annotated perf event, or <code>null</code> if the resulting event isn't deemed worthy
+     *     of reporting.
+     */
+    @Nullable
+    @CheckReturnValue
+    public static PerfEvent populatePerfEventFromLogger(
+            ComponentContext c, ComponentsLogger logger, @Nullable PerfEvent perfEvent) {
+      return populatePerfEventFromLogger(c, c.getTreeProps(), logger, perfEvent);
```

Comment:
I suggest just passing `null` here and logic of the main method will decide what to use as a fallback

Reply:
Thanks, fixed.